user_guide/common_features.rst

```diff
@@ -0,0 +1,79 @@
+.. _user-guide-common-features:
+
+Common Features
+===============
+
+Customers, Channels and Accounts
+--------------------------------
+
+One of the main tasks for any CRM is ability to aggregate the details of a customer. 
+For example, you have two web-based stores, a retail outlet and a factory. There is a company whose managers 
+purchase from your web-based stores and with whom you have some business arrangements at the 
+factory. 
+
+Of course, there are much more details and peculiarities, but if we keep it simple, in an on-line store, somebody comes 
+to the store's website, looks through the items offered, and if interested in 
+buying them puts the items in the cart and orders them, leaving you the contact and shipping details. In many cases the 
+web-store customers can log-in and their profile details will be used. What kind of information do you need to make 
+relations with such customers successful? The most basic things are - what pages have people viewed, what items they 
+have ordered, if there is anything that a customer has added to the cart but hasn't ordered and if there are some 
+complaints after the delivery.
+
+It is quite different when you are to execute an agreement in your office, there may be some complex arrangements, 
+details whereof you want to track, some preliminary agreements and executed deeds. You can have several ongoing and 
+perspective projects with one big company, but with different project managers, so you need their contact details, as 
+well as understanding of their position in the company and relation to the project. 
+
+On one hand, you want to keep information from different sources allocated, but at the same time, for comprehensive 
+customer relations, you need to get the full view of the company's purchase activity.
+
+In OroCRM this is done with the three types of records: channels, customers and accounts. 
+
+**Channels** are sources of the information (in our example of the web-stores and the company) are 
+Channel records. Having different channels for each source is very convenient, as you can specify what details will be 
+collected for customers created within each of the channels.
```

Comment:
Edit: **Channels** are the various sources of customer information. For instance, in our previous example of a company with a retail story and a factory, both would be different channels. It can be convenient having different channels for each source, as you will better be able to control what specific details will be collected for each customer.